lips/lip-3.md

```diff
@@ -8,52 +8,51 @@ created: 06/09/2020
 ---
 
 # Introduction
-Libra Upgrade Management (LUM) is a set of protocols for the Libra Networks (LN) company to reconfigure the blockchain. With respect to time-sensitive decisions and incident response, LUM provides the Libra Networks--a subsidiary of the Libra Association licensed in Switzerland to operate a payment system--with technical means to respond to critical incidents, as (i) necessitated by its license terms and the law and (ii) as determined by the Libra Network risk appetite/tolerance policy.
-
-It should be noted that the Association appoints LN officers. Officers are accountable for their actions, and usual measures may be taken upon deviation from their expected conduct.  Hence, the ultimate responsibility and power is by distributed governance of the Association.
+Libra Upgrade Management (LUM) is a set of protocols to reconfigure the blockchain. With respect to time-sensitive decisions and incident response, LUM provides the Libra Networks - a subsidiary of the Libra Association licensed in Switzerland to operate a payment system - with technical means to respond to critical incidents, as necessitated by compliance.
 
 Due to being a reserve backed currency, the true blockchain is the one that is maintained by the Libra Association and reflects the assets in the Libra Reserve. In other words, in contrast to Bitcoin and other “proof of work” and “proof of stake” networks where the miners and users beliefs determine the current fork of the blockchain, with Libra the blockchain used by the Libra Reserve determines the true blockchain. Furthermore, Libra Networks operates as a regulated payment company under a Swiss license and must be able to enforce its policy and oversight, as well as to respond to critical operational incidents.
 
-The goal of LUM is to provide the Association with means to exercise its authority to reconfigure the blockchain and to effectively remediate any plausible operational incident while maintaining the trust gained through decentralized governance and distributed systems.
+The goal of LUM is to provide the means to effectively and transparently update the software in order to meet changing requirements as well as remediate any plausible operational incident.
+
 We first present all building blocks and the reasoning, followed by concrete examples.
 
 # Building blocks
-There’s two types of upgrades in the system - (i) updates to the Libra protocol software and (ii) changes to on-chain state. We discuss them correspondingly.
+There are two types of upgrades in the system - (i) [updates to the Libra protocol software](#software-upgrade) and (ii) [changes to on-chain state](#on-chain-state-upgrade). We discuss them correspondingly.
```

Comment:
I propose the following start:


Conceptually, the software running on validator nodes comprises two layers. One is a *blockchain core* whose goal is to replicate a ledger of transactions and their outcome. The second is a *framework* that stores the ledger-state and a set of software modules  -- encoded in the Move programming language -- that define the rules for interacting with the LPN and mutating the ledge-state (see also LIP-2[ref]). There are two corresponding type of upgrades to the system, (i) updates to the core software (ii) updates to the framework state